leku/src/main/res/layout-land/activity_location_picker.xml

```diff
@@ -112,6 +113,37 @@
             >
 
           <TextView
```

Comment:
Yeah, but you also didn't do it that way with the other address views.. so that was why i also didn't do it...?